utils/python/CIME/XML/machines.py

```diff
@@ -235,3 +235,33 @@ def get_batch_system_type(self):
         """
         batch_system = self.get_node("batch_system")
         return batch_system[0].get("type")
+
```

Comment:
Curious as to your opinion here. I was trying to follow the system being followed in the rest of the file.